sched/generators.go

```diff
@@ -2,15 +2,28 @@ package sched
 
 import (
 	"fmt"
+	"math/rand"
+	"time"
+
 	"github.com/leanovate/gopter"
 	"github.com/scootdev/scoot/runner"
-	"github.com/scootdev/scoot/sched/gen-go/schedthrift"
 	"github.com/scootdev/scoot/tests/testhelpers"
-	"time"
 )
 
 func GenJob(id string, numTasks int) Job {
-	jobDef := GenJobDef(numTasks)
+	rand := testhelpers.NewRand()
```

Comment:
The naming of GenJob and GenRandomJob is a little confusing at first. What's the difference between NewRand() and genParams.Rng and can these funcs be combined?